### DISCUSSION TITLE
InfoWindow Hover

### DISCUSSION THREAD

**Discussion by kevingerstner** [visgl/react-google-maps#196] [Ideas]
It would be great to have an option to show InfoWindows on hover, similar to apps like Yelp or Tripadvisor. Is this a limitation because of the close button that cannot be controlled by the API? If so, is it possible to create a custom infowindow?

**Comment by usefulthink** [comment]
You can still use the InfoWindow component for this, since it could be automatically shown on hover and it will simply disappear when it's no longer rendered in the react-component tree. I think there is also be a hacky way to remove the 'x'-button from the html in these cases using some css.

Alternatively, you can use e.g. the advanced-markers to add html content to the map at a specified lat/lng position. See here for an example: https://developers.google.com/maps/documentation/javascript/advanced-markers/html-markers. (when using this library you can pass the marker html-content to the `<AdvancedMarker>` as child-components, [see here](https://github.com/visgl/react-google-maps/blob/11fdc133cd526699d4f965008a87b8b4f4bc330a/examples/markers-and-infowindows/src/app.tsx#L57-L73)).
